app/com/arpnetworking/metrics/portal/reports/impl/chrome/DevToolsServiceWrapper.java

```diff
@@ -131,6 +133,9 @@ public Object evaluate(final String js) {
 
     @Override
     public void close() {
+        if (_closed.getAndSet(true)) {
+            return;
```

Comment:
Description states any other calls are illegal after close, but does not behave any differently. Should we throw ISE if closed on any method call?